### ISSUE TITLE
Wrong axisRange placement if added later to the chart

### ISSUE THREAD

**Issue by lticadler** [amcharts/amcharts4#3137]
**Bug description**

In my scenario users could add visual marks in the chart and show/hide them.

If an AxisRange is added invisible (grid.disabled=true) to a chart some time after the chart is created and then again a bit later changed to be visible the position is wrong.

See https://codepen.io/chrisadler/pen/WNGbVEZ?editors=1010

From the pen:
1000 ms timeout (wrong position):
![image](https://user-images.githubusercontent.com/58069494/100600695-ea128c00-3301-11eb-98c4-9497e025768f.png)

1ms timeout (correct possition):
![image](https://user-images.githubusercontent.com/58069494/100600774-06162d80-3302-11eb-8e03-7563958384de.png)

**Environment (if applicable)**

 - amCharts version: 4.10.12
 - Browser and its version: all Browsers

**Comment by martynasma** [comment]
I suggest you use `visible` instead of `disabled`.

```
axisRange.grid.visible = false;
```

`visible` is better because it sets elements visibility but does not remove from the layout, like `disabled` does.

**Comment by lticadler** [comment]
Thanks vor the advice.
I updated the pen with a second axisrange for comparison.

**Comment by lticadler** [comment]
Now I remember why I used **disabled**:
I use AxisRange on both axis and handled everything in the same way (showing/hiding the ranges).

The valueaxis should include ranges: `valueAxis.includeRangesInMinMax = true;`
Therefore for value axis axisranges the `ignoreMinMax` flag is set (dynamically) in order to show the range in the chart although the series values aren't in the range.` (e.g. series value 10 but axis range 50 -> value axis from 0 to 60 )`.
By enabling/showing or disabling/hiding the range the value axis should rescale ` (range disabled -> value axis from 0 to 20)`.
The rescaling doesn't work when using the suggested visible flag instead of the disabled flag.

Now I use the visible flag for the date axis ranges and the disabled flag for the value axis ranges.

Again I updated the pen and added 2 value axis ranges.
As I said - the one using the visible flag does not work. Don't know it that should be considered a bug?
